builtin-modules.json

```diff
@@ -21,23 +24,30 @@
 	"net",
 	"os",
 	"path",
+	"path/posix",
+	"path/win32",
 	"perf_hooks",
 	"process",
 	"punycode",
 	"querystring",
 	"readline",
+	"readline/promises",
 	"repl",
 	"stream",
+	"stream/consumers",
+	"stream/promises",
+	"stream/web",
 	"string_decoder",
 	"timers",
+	"timers/promises",
 	"tls",
 	"trace_events",
 	"tty",
 	"url",
 	"util",
+	"util/types",
 	"v8",
 	"vm",
-	"wasi",
 	"worker_threads",
```

Comment:
I don't know why `wasi` was here, because even before my changes, running `npm run make` did not output it (tested in Node.js 6, 8, 10, 12, 14, 16, 17, 18). After some investigation I found that it is a module hidden under an experimental flag. If this flag is set, then this module will include it. However, I think it's better to omit it from the static list, since usually it will not be present.